src/Carousel/carouselHelper.tsx

```diff
@@ -725,6 +725,39 @@ export default function carouselHelper(settings: CarouselSettings) {
     }
 
     let destinationIndex = dir
+
+    const controlButtonLogic = () => {
```

Comment:
A very pedantic suggestion of mine, but I suggest renaming the `controlButtonLogic` function name to something more descriptive that tells us what this function is doing. Its pretty tough deciding what this one should be named, but I think it should be named `handleControlButtonSlideShift` ... Also, can we define the function outside of the `shiftSlide` function itself?